dashboard/test/models/pd/enrollment_test.rb

```diff
@@ -426,7 +426,7 @@ class Pd::EnrollmentTest < ActiveSupport::TestCase
 
   test 'get safe names' do
     enrollments = create_list :pd_enrollment, 5
-    safe_names = Pd::Enrollment.where(id: enrollments.map(&:id)).order(:id).get_safe_names
+    safe_names = Pd::Enrollment.where(id: enrollments.pluck(:id)).order(:id).get_safe_names
```

Comment:
Note, the result of create_list is an array of populated (and persisted) ActiveRecord objects, so again `pluck` doesn't save anything here (though it doesn't hurt).